tests/security/apparmor_profile/apache2_changehat.pm

```diff
@@ -149,6 +149,14 @@ sub run {
         record_info("ERROR", "There are denied profile_replace records found in $audit_log", result => 'fail');
         $self->result('fail');
     }
+    # Due to bsc#1191684, add following check points as well
+    my @check_list = ('file_receive', 'open', 'signal', 'mknod');
+    foreach my $check_point (@check_list) {
+        if ($script_output =~ m/type=AVC .*apparmor=.*DENIED.* operation=.*$check_point.* profile=.*httpd-prefork.*/sx) {
```

Comment:
maybe you could try if `$script_output =~ m/type=AVC .*apparmor=.*DENIED.* operation=.*(file_receive|open|signal|mknod).* profile=.*httpd-prefork.*/sx` make sense.

Reply:
Thanks Xiaojing,

I think the current logic is list all failed point if we use loop check :)